template.yaml

```diff
@@ -1081,6 +1081,7 @@ Resources:
       Environment:
         Variables:
           ALLOWED_ORIGIN: !Ref AllowedOrigins
+          TABLE_NAME: !Ref NvaResourcesTable
```

Comment:
Why do we need to add this environment variable now? I see there are already references to it in old code (such as `UpdateImportStatusHandler`), so I'm a bit confused.

Reply:
I have updated body of this handler. It returned "hardcoded" response object with empty list before and did no database interactions. Now it reads entries from database, without providing table name it fails on execution.